pkg/obs/services/computes.go

```diff
@@ -0,0 +1,167 @@
+package services
+
+import (
+	"fmt"
+	"github.com/huaweicloud/huaweicloud-csi-driver/pkg/config"
+	"github.com/huaweicloud/huaweicloud-sdk-go-obs/obs"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func GetBucket(c *config.CloudCredentials, bucket string) (string, error) {
+	exists, err := CheckBucketExists(c, bucket)
+	if err != nil || !exists {
+		return "", err
+	}
+	if enabled, e := CheckBucketFSStatus(c, bucket); !enabled {
+		if e != nil {
+			return "", e
+		}
+		return "", status.Errorf(codes.Unavailable, "Error, OBS instance %s is not FSFileInterface", bucket)
```

Comment:
`FSFileInterface` is not easy to understand, it is recommended to change to: 
`Error, the OBS bucket %s is not a parallel file system`

Reply:
done